libknet/bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "rust-kronosnet"
```

Comment:
this file should be autogenerated by ./configure (I can do it, no worries about that).

Can the name be kronosnet ?

Reply:
the name can be anything that is not yet taken on crates.io ;)

some conventions are 

- $foo-sys or lib$foo-sys for generated FFI bindings
- $foo-internal/impl for unstable, implementation detail stuff not for external consumption, but split out for some reason
- $foo or rust-$foo for the main crate
- $foo-$bar if the crate gets so big that it makes sense to split it into multiple distinct reusable parts